files/scripts/write_standby.py

```diff
@@ -144,19 +157,23 @@ def apply_mux_config(self):
             logger.log_warning("Skip setting mux state due to ongoing warmrestart.")
             return
 
-        intfs = self.get_auto_mux_intfs()
-        state = 'standby'
+        modes = self.get_all_mux_intfs_modes()
         if self.wait_for_tunnel():
-            logger.log_warning("Applying {} state to interfaces {}".format(state, intfs))
+            logger.log_warning("Applying state to interfaces {}".format(modes))
             producer_state_table = ProducerStateTable(self.appl_db, 'MUX_CABLE_TABLE')
-            fvs = create_fvs(state=state)
 
-            for intf in intfs:
+            for intf, state in modes.items():
+                fvs = create_fvs(state=state)
                 producer_state_table.set(intf, fvs)
         else:
             logger.log_error("Timed out waiting for tunnel {}, mux state will not be written".format(self.tunnel_name))
 
 
 if __name__ == '__main__':
-    mux_writer = MuxStateWriter()
+    parser = argparse.ArgumentParser(description='Write initial mux state')
+    parser.add_argument('-s', '--state',
+                        help='state: intial state for "auto" and/or "manual" config, default "standby"',
+                        type=str, required=False, default='standby')
+    args = parser.parse_args()
```

Comment:
i think we've few places where write_standby script is used/planned to use.  So just want to confirm that we tested the case if no param given, it is default to 'standby'.

Reply:
yes. I tested that.